fgspectra/frequency.py

```diff
@@ -176,29 +147,53 @@ def __call__(self, nu, nu_0):
         return ThermalSZ.f(nu) / ThermalSZ.f(nu_0)
 
 
-class UnitSED(SED):
+class ConstantSED(Model):
     """Frequency-independent component."""
 
-    def __call__(self, nu, *args):
-        return np.ones_like(np.array(nu))
+    def eval(self, nu=None, amp=1.):
+        amp = np.array(amp)
+        return amp * np.ones_like(np.array(nu)) * _rj2cmb(nu)
+
 
+class Join(Model):
+    """ Join several SED models together
+    """
 
-class Join(SED):
+    def __init__(self, *seds, **kwargs):
+        """ Join several SED models together
 
-    def __init__(self, *seds):
+        Parameters
+        ----------
+        *sed:
+            Sequence of SED models to be joined together
+        """
         self._seds = seds
 
-    def __call__(self, kwargs_seq=None):
+    def set_defaults(self, **kwargs):
+        if 'kwseq' in kwargs:
+            for sed, sed_kwargs in zip(self._seds, kwargs['kwseq']):
+                sed.set_defaults(**sed_kwargs)
+
+    def _get_repr(self):
+        return {type(self).__name__: [sed._get_repr() for sed in self._seds]}
+
+    @property
+    def defaults(self):
+        return {'kwseq': [sed.defaults for sed in self._seds]}
+
+    def eval(self, kwseq=None):
```

Comment:
Previously we had `kwargs_seq`, I changed it to `kwseq`...just becuase it is shorter and easier to type. Is it less clear? I didn't propagate everywhere this change because I'm afraid it is